### ISSUE TITLE
PropHunt and Class Warfare Bugs

### ISSUE THREAD

**Issue by unusual917** [Scags/TF2-Jailbreak-Redux#26]
Hi Scags, some bugs for Prophunt. I used !flr since I was testing with a bot.
1) Won't show on menu.
![alt text](http://icecream.me/d5ba24756e8fc1ec7370e53510926a7c)
direct image link: http://icecream.me/d5ba24756e8fc1ec7370e53510926a7c
2) Red player spawns with normal player model instead of becoming a prop.

Class Warfare still has the no weapons bug but Warday has been fixed.
Also any chance we can allow engineers to build buildings in class warfare? ~~Same with saxton hale LR (if knockback stuff is implemented that is? (nvm sentries on saxton might be too hard).~~

I used the latest compiled smx uploaded on git.

**Comment by Scags** [comment]
For 1 I'm guessing you might have disabled the plugin via cvar or you reloaded it mid-round (which should be handled anyways?).  I wasn't having this issue just the other day. It could essentially be a variety of things, so I guess update SourceMod to the last 1.9 build. 
For 2 that was an issue I thought that I'd fixed, are they A-posed scouts? If so then that be a problem.
Class Warfare and Warday are set up the exact same way, so that shouldn't happen? I have a hunch but if it persists try updating SourceMod.

**Comment by unusual917** [comment]
Hey Scags, so I did some thorough testing and seems like everything is working for the most part! I updated sourcemod and that was the issue.

I had a few concerns after testing stuff out.
3) Is it normal on warday, classwars, and saxton hale for engies to not be able to build sentries and other buildings? Not sure why on classwars it lets my model carry the building box (looks like a lunchbox) but it doesn't give me a placement hologram or let me build anything. For warday though, it doesn't even give me the lunchbox.

4) Saxton hale LR allows heavies to equip Natascha that gives incredible knockback making it near impossible for hale to win. Is there a way to decrease knockback for this? I tried adding attribute 32 on line 1767 but didn't work.

As usual, thanks for the help!
